src/internal/abstraction.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  SimpleTransaction,
+  InputGenerateTransactionOptions,
+  TypeTagAddress,
+  TypeTagStruct,
+  stringStructTag,
+} from "../transactions";
+import { AccountAddressInput } from "../core";
+import { generateTransaction } from "./transactionSubmission";
+import { MoveFunctionId } from "../types";
+import { AptosConfig } from "../api/aptosConfig";
+import { getFunctionParts } from "../utils/helpers";
+
+export async function addDispatchableAuthenticationFunctionTransaction(args: {
+  aptosConfig: AptosConfig;
+  sender: AccountAddressInput;
+  authenticationFunction: string;
+  options?: InputGenerateTransactionOptions;
+}): Promise<SimpleTransaction> {
+  const { aptosConfig, sender, authenticationFunction, options } = args;
+  const { moduleAddress, moduleName, functionName } = getFunctionParts(authenticationFunction as MoveFunctionId);
+  return generateTransaction({
+    aptosConfig,
+    sender,
+    data: {
+      function: "0x1::account_abstraction::add_dispatchable_authentication_function",
```

Comment:
I removed `_dispatchable` in the move function.
please update the 3 places in this file.